hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/KeyDeletingService.java

```diff
@@ -172,24 +129,9 @@ public BackgroundTaskResult call() throws Exception {
           List<BlockGroup> keyBlocksList = manager
               .getPendingDeletionKeys(keyLimitPerTask);
```

Comment:
Despite the PR title, I don't see the code to make KDS aware of snapshots is implemented yet in this PR? In order to do that, extra logic has to be added in `OmMetadataManagerImpl#getPendingDeletionKeys()` to check whether the key exists in the previous snapshot (in the same scope).

Reply:
Thanks for pointing that out. As per our discussion offline, I have updated the PR.